client/src/components/ProjectSamplesTable.js

```diff
@@ -78,7 +78,9 @@ export const ProjectSamplesTable = ({
     { Header: 'Technology', accessor: 'technologies' },
     {
       Header: 'Other Modalities',
-      accessor: ({ modalities }) => modalities || 'N/A'
+      accessor: ({ modalities }) => (
+        <>{modalities.length ? <>{modalities.join(', ')}</> : 'N/A'}</>
```

Comment:
```suggestion
        <>{modalities.length ? modalities.join(', '): 'N/A'}</>
```

doesn't this work all the same?

Reply:
Complented, thank you!